src/test/java/com/deque/html/axecore/selenium/AxeBuilderUnitTest.java

```diff
@@ -105,10 +105,10 @@ public void shouldHandleIfOptionsAndContextNotSet()
     AxeBuilder builder = new AxeBuilder();
     Results result = builder.analyze(this.webDriver);
     verifyAxeResultsNotNull(result);
-    Assert.assertEquals(42, result.getInapplicable().size());

```

Comment:
We should fix these tests, just assert the size isn't 0 or something, so we don't have to change these numbers every update.